src/payment/coupon/CouponForm.jsx

```diff
@@ -70,8 +42,7 @@ export class CouponForm extends Component {
       <form onSubmit={this.handleAddSubmit} className="mb-3 d-flex align-items-end">
         <ValidationFormGroup
           for={id}
-          invalid={errorCode !== null}
-          invalidMessage={this.renderInvalidMessage()}
+          invalid={false}
```

Comment:
We don't want contextual messages here anymore.  Thus a bunch of stuff gets deleted.